changelogs/fragments/86074-None-value-in-required_if.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Update validation logic for parameter checks to allow for None value for required_if
```

Maybe this could be a little more specific that the issue occurred for omitted options, something like:

```suggestion
  - ``required_if`` - fix argument validation for value ``None`` when the option is omitted and defaults to ``None``.
```